osu.Game/Screens/Play/MasterGameplayClockContainer.cs

```diff
@@ -56,6 +57,11 @@ public class MasterGameplayClockContainer : GameplayClockContainer, IBeatSyncPro
         /// </summary>
         private double? actualStopTime;
 
+        /// <summary>
+        /// Maintained solely to delegate <see cref="IAdjustableAudioComponent"/> pieces to (to maintain parent lookups).
+        /// </summary>
+        private readonly AudioContainer audioContainer;
```

Comment:
Is there a reason this is an `AudioContainer` instead of `AudioAdjustments`?

Reply:
`AudioContainer` will correctly lookup parent adjustments. If I was to create an isolated instance of `AudioAdjustments` here, any inherited information would be lost at the point of `GameplayClockContainer`.

Reply:
In case the reasoning here is unclear, this stems from the fact that making `MasterGameplayClockContainer` inherit from `IAdjustableAudioComponent`, it is seen in the hierarchical check for parent adjustments.

Your proposed change would avoid this.